adala/runtimes/_openai.py

```diff
@@ -382,11 +387,13 @@ async def batch_to_batch(
                     if name in options:
                         completion_text = match_options(completion_text, options[name])
                     outputs.append({name: completion_text})
+                    index.append(idx)
```

Comment:
can we just include the index in the `outputs` so we don't have to manage a separate iterable? So just do something like `outputs.append({name: completion_text, "index": idx})`